eZ/Publish/Core/MVC/Symfony/FieldType/Tests/User/ParameterProviderTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\MVC\Symfony\FieldType\Tests\User;
+
+use DateInterval;
+use DateTime;
+use eZ\Publish\API\Repository\UserService;
+use eZ\Publish\API\Repository\Values\Content\Field;
+use eZ\Publish\API\Repository\Values\User\User;
+use eZ\Publish\Core\FieldType\User\Value;
+use eZ\Publish\Core\MVC\Symfony\FieldType\User\ParameterProvider;
+use PHPUnit\Framework\TestCase;
+
+class ParameterProviderTest extends TestCase
+{
+    private const EXAMPLE_USER_ID = 1;
+
+    /** @var \eZ\Publish\API\Repository\UserService|\PHPUnit\Framework\MockObject\MockObject */
+    private $userService;
+
+    /** @var \eZ\Publish\API\Repository\Values\User\User|\PHPUnit\Framework\MockObject\MockObject */
+    private $user;
+
+    /** @var \eZ\Publish\Core\MVC\Symfony\FieldType\User\ParameterProvider */
+    private $parameterProvider;
+
+    protected function setUp(): void
+    {
+        $this->userService = $this->createMock(UserService::class);
+        $this->user = $this->createMock(User::class);
+        $this->parameterProvider = new ParameterProvider($this->userService);
+    }
+
+    public function testGetViewParameters(): void
+    {
+        $passwordExpiresIn = 14;
+        $passwordExpiresAt = (new DateTime())->add(new DateInterval('P14D'));
+
+        $this->userService
```

Comment:
this fragment could be extracted to private method 
```
        $this->userService
            ->method('loadUser')
            ->with(self::EXAMPLE_USER_ID, [])
            ->willReturn($this->user);
```

Reply:
Moved to the `setUp` method 😉